setup.py

```diff
@@ -1,21 +1,54 @@
+import codecs
+import os
+import re
+
 from setuptools import setup, find_packages
 
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+def read(*parts):  # Stolen from txacme
+    with codecs.open(os.path.join(HERE, *parts), 'rb', 'utf-8') as f:
+        return f.read()
+
+
+def readme():
+    # Prefer the ReStructuredText README, but fallback to Markdown if it hasn't
+    # been generated
+    if os.path.exists('README.rst'):
+        return read('README.rst')
+    else:
+        return read('README.md')
+
+
+def get_version(package):
```

Comment:
In the past we've preferred to use a `VERSION` file rather than `__init__.py` as seen [here](https://github.com/praekelt/mc2/blob/develop/setup.py#L15-L19).

Is there any benefit to putting the version in __init__.py?